pyigm/surveys/llssurvey.py

```diff
@@ -250,6 +250,7 @@ def load_HDLLS(cls, load_sys=True, grab_spectra=False, isys_path=None):
 
         # Set data path and metallicity
         spath = pyigm_path+"/data/LLS/HD-LLS/Spectra/"
+        lls_survey.build_all_abs_sys()
```

Comment:
did you add this @marijana777 ?

I don't think we want this here.

Reply:
I did not, as I know